packages/bundler-vite/src/build/build.ts

```diff
@@ -72,21 +72,20 @@ export const build = async (
     const { app: vueApp, router: vueRouter } = await createVueApp()
     const { renderToString } = await import('vue/server-renderer')
 
+    if (spinner) spinner.text = `Starting to render pages ${chalk.magenta(app.pages?.map(vuePressPage => vuePressPage.path))}`
```

Comment:
I dislike the idea here, I would choose to remove page path, for my theme docs around 800 pages, if would be annoying with tons of path throwing to logs without even a line break.

Reply:
> I dislike the idea here, I would choose to remove page path, for my theme docs around 800 pages, if would be annoying with tons of path throwing to logs without even a line break.
```js
if (spinner) spinner.text = `Starting to render pages ${chalk.magenta(app.pages.map(vuePressPage => vuePressPage.path)) + '\n'}`
```

Like this ?

Reply:
\n

Reply:
> I dislike the idea here, I would choose to remove page path, for my theme docs around 800 pages, if would be annoying with tons of path throwing to logs without even a line break.

@Mister-Hope  Sorry, is current change is ok ?